ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+New deprecations
+   * Deprecate MBEDTLS_SSL_SRV_SUPPORT_SSLV2_CLIENT_HELLO that enables parsing
+     SSLv2 ClientHello messages to interoperate with old SSLv2 servers.
```

Comment:
I find the "to interoperate with old SSLv2 servers" part of the sentence confusing: parsing a ClientHello clearly doesn't help interop with servers - it helps interop with clients who want to interop with SSLv2 servers. I'd just drop that part of the sentence. I don't think we need to be too descriptive here - the fact the the option name contains SSLV2 should be more than enough justification.